js/components/wysiwyg.js

```
@@ -39,7 +39,7 @@ Fliplet.FormBuilder.field('wysiwyg', {
   watch: {
     value: function(val) {
       // This happens when the value is updated programmatically via the FormBuilder field().val() method
-      val = _.isNumber(val) ? _.toString(val) : val;
+      val = typeof val === 'number' ? val.toString() : val;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)